### PR TITLE
Add `Content-Type: application/json` for fiber requests

### PR DESCRIPTION
**Pull request by afbora**
### Fixes

- Fixes blocking requests by ModSecurity (owasp rules)

### Breaking changes
None

## Ready?

- [x] Unit tests for fixed bug/feature
- [x] In-code documentation (wherever needed)
- [x] Tests and checks all pass

### For review team

- [x] Add changes to release notes draft in Notion
- [x] Add to [website docs release checklist](https://github.com/getkirby/getkirby.com/pulls) (if needed)

### CODE REVIEW

**Review by lukasbestle — DISMISSED** [review]

File: panel/src/fiber/index.js [hunk 1]
```diff
@@ -235,6 +235,7 @@ export default class Fiber {
 				cache: "no-store",
 				headers: {
 					...this.options.headers(),
+					"Content-Type": "application/json",
```
Comment (afbora) [inline panel/src/fiber/index.js]: @lukasbestle Does it matter if it is lowercase or not?

Comment (afbora) [reply panel/src/fiber/index.js]: I just wonder because it's lowercased in `api::request()`.

Comment (lukasbestle) [reply panel/src/fiber/index.js]: [MDN says](https://developer.mozilla.org/en-US/docs/Web/API/Headers) that `fetch` handles headers case-insensitively, so I guess it doesn't matter. The same applies to the HTTP spec itself.

Comment (afbora) [reply panel/src/fiber/index.js]: ActualIy I mean for overwriting content type.

Comment (lukasbestle) [reply panel/src/fiber/index.js]: I just tested it to make sure:

```js
fetch('/', {
  headers: {
    "Content-Type": "application/json",
    "content-type": "text/something"
  }
});
```

In Safari 15.6.1, `fetch()` merges the two unfortunately, so the request header becomes `application/json, text/something`.

That's indeed not great as it means that the headers are treated *partly* case-sensitively.

Comment (afbora) [reply panel/src/fiber/index.js]: What do you suggest? 

- Always use lowercase?
- Is it always to lowercase the keys before merging them? 

Do you have another idea?

Comment (lukasbestle) [reply panel/src/fiber/index.js]: > Is it always to lowercase the keys before merging them?

I think that's a great idea!

Comment (distantnative) [reply panel/src/fiber/index.js]: We should do it then in `request.js` as well

Comment (afbora) [reply panel/src/fiber/index.js]: Done ✅

File: panel/src/api/request.js [hunk 2]
```diff
@@ -23,7 +25,7 @@ export default (config) => {
 				headers: {
 					"x-requested-with": "xmlhttprequest",
```
Comment (afbora) [inline panel/src/api/request.js]: @distantnative Do we need `x-requested-with` for `fiber.js` too?

Comment (bastianallgeier) [reply panel/src/api/request.js]: We used this initially to detect fiber requests, but as far as I know, it's no longer needed.

Comment (afbora) [reply panel/src/api/request.js]: 👍

**Review by bastianallgeier — DISMISSED** [review]
Looks good to me. Do you want to remove the x-requested-with header or do we keep it?

**Comment by afbora** [comment]
I thought it wasn't needed in just fiber.js. I've removed ✅

**Review by bastianallgeier — APPROVED** [review]

**Pull request merged**